### PR TITLE
Fix Gingko style bugs

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd

### CODE REVIEW

File: lms/static/sass/views/_login-register.scss [hunk 1]
```diff
@@ -78,10 +78,11 @@
     $grid-columns: 12;
     background: $white;
     min-height: 100%;
+    margin: 0 auto;
     padding-left: ($baseline/2);
     padding-right: ($baseline/2);
     $third-party-button-height: ($baseline*1.75);
-    display: inline-block;
+    display: block;
```
Comment (stvstnfrd) [inline lms/static/sass/views/_login-register.scss:85]: We really shouldn't be forking platform styling for this.
Particularly, it seems odd that we would need/want to touch login/register pages.

Why this change?

File: lms/static/sass/stanford/course/_sneakpeek_courseware.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.sneakpeek-warning {
```
Comment (stvstnfrd) [inline lms/static/sass/stanford/course/_sneakpeek_courseware.scss:1]: Can we combine these two files?
Worst case, we would need to wrap this block with a parent selector, like done above in `about` via:

```sass
.course-info {
    .sneakpeek-warning {
        # ....
    }
}
```

But that may not even be necessary. Then there's just one file to worry about..

**Review by stvstnfrd — CHANGES_REQUESTED** [review]

File: lms/static/sass/stanford/course/_sneakpeek_courseware.scss (hunk 2 above)
Comment (caesar2164) [reply lms/static/sass/stanford/course/_sneakpeek_courseware.scss:1]: We can't, because the course overview page (the new page that is the issue in the first place for sneak-peek) does NOT have a body class to grab on to.

File: lms/static/sass/views/_login-register.scss (hunk 1 above)
Comment (caesar2164) [reply lms/static/sass/views/_login-register.scss:85]: It's necessary to make the centering work, also upstream already has this change, that's why the upstream PR I created only adds the margin.

https://github.com/edx/edx-platform/pull/18189

File: lms/static/sass/features/_course-experience.scss [hunk 3]
```diff
@@ -203,3 +203,5 @@
 .course-reviews-tool {
   margin: ($baseline * 2) ($baseline * 3);
 }
+
+@import '../stanford/_sneakpeek_courseware.scss';
```
Comment (stvstnfrd) [inline lms/static/sass/features/_course-experience.scss]: Why the new include?

Comment (caesar2164) [reply lms/static/sass/features/_course-experience.scss]: cause this is a new SASS file that styles the new course overview page.

File: lms/static/sass/stanford/course/_sneakpeek_courseware.scss [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    position: relative;
+    max-width: 1170px;
+    margin: 0 auto;
+    padding: 5px;
+    border: none;
+    background-color: #000000;
```
Comment (stvstnfrd) [inline lms/static/sass/stanford/course/_sneakpeek_courseware.scss]: Why is this hard-coded to a hex value, but the color immediately below set to a variable?
And if we're hard-coding, why not the 3-char value? `#000`?

File: lms/static/sass/stanford/course/_sneakpeek_courseware.scss [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    color: $white;
+    line-height: 2em;
+    font-weight: bold;
+    text-align: center;
+}
+.sneakpeek-warning a {
+    padding: 5px;
+    background-color: #FFFFFF;
```
Comment (stvstnfrd) [inline lms/static/sass/stanford/course/_sneakpeek_courseware.scss]: Same question, why hard-coded?

**Review by stvstnfrd — APPROVED** [review]
+1. squash but do _not_ merge.

I'll have to incorporate this into my merge script, so post-squash, let me know which SHAs to keep.

**Comment by caesar2164** [comment]
these last 2 commits, @stvstnfrd ^

File: lms/templates/courseware/tab-view.html [hunk 6]
```diff
@@ -35,3 +35,5 @@
 <%block name="content">
     ${HTML(fragment.body_html())}
 </%block>
+
+<%include file="sneak_peek_tos_privacy.html" args="user=user, course=course" />
```
Comment (stvstnfrd) [inline lms/templates/courseware/tab-view.html:39]: Screenshots before/after please.

Also, did you test this with sneak peek _disabled_?

Comment (caesar2164) [reply lms/templates/courseware/tab-view.html:39]: Before:
screen shot 2018-05-16 at 1 23 46 pm

After:
screen shot 2018-05-16 at 1 23 26 pm

Yes and it works fine because the conditional is in that included file...
Also, you can't reach any of the course pages if sneak peek is disabled...

**Comment by stvstnfrd** [comment]
pushed to my branch
